Jenkinsfile.github

```diff
@@ -2,7 +2,7 @@
 
 pipeline {
     agent {
-        label "metal-gcp-builder"
+        label "metal-gcp-builder-new"
```

Comment:
For this POC, I had to create new configuration in Jenkins, which features git vendor extension. This particular change is temporary, we'll rollback to "metal-gcp-builder" once main configuration is updated.